### PR TITLE
Add all_points rpc query and benchmark for streaming points

### PR DESCRIPTION
**Pull request by nnmm**
Useful for performance optimization. The GetAllPoints query is helpful because it's a common use case and just using a huge bounding box is hacky. Furthermore, in the benchmark we don't know which pointcloud, and thus bounding box, will be used.

### CODE REVIEW

File: point_viewer_grpc/src/bin/octree_benchmark.rs [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        ])
+        .get_matches();
+
+    let octree_directory = PathBuf::from(
+        matches
+            .value_of("octree_directory")
+            .expect("octree_directory not given"),
+    );
+    let num_points = u64::from_str(matches.value_of("num-points").unwrap_or("50000000"))
+        .expect("Could not num-points");
```
Comment (tblaette) [inline point_viewer_grpc/src/bin/octree_benchmark.rs]: nit: give a proper error message

File: point_viewer_grpc/src/bin/octree_benchmark.rs [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    if matches.is_present("no-client") {
+        server_benchmark(octree_directory, num_points)
+    } else {
+        let port = value_t!(matches, "port", u16).unwrap_or(50051);
+        full_benchmark(octree_directory, num_points, port)
+    }
+}
+
+fn server_benchmark(octree_directory: PathBuf, num_points: u64) {
+    let octree = OnDiskOctree::new(&octree_directory).unwrap();
```
Comment (tblaette) [inline point_viewer_grpc/src/bin/octree_benchmark.rs]: do an `.expect("proper error message")` instead of `.unwrap()`

Comment (nnmm) [reply point_viewer_grpc/src/bin/octree_benchmark.rs]: Done

File: point_viewer_grpc/src/bin/octree_benchmark.rs (hunk 1 above)
Comment (nnmm) [reply point_viewer_grpc/src/bin/octree_benchmark.rs]: Done

File: point_viewer_grpc/src/bin/octree_benchmark.rs [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    let env = Arc::new(Environment::new(1));
+    let ch = ChannelBuilder::new(env).connect(&format!("localhost:{}", port));
+    let client = OctreeClient::new(ch);
+
+    let req = proto::GetAllPointsRequest::new();
+    let receiver = client.get_all_points(&req).unwrap();
+
+    let mut counter: u64 = 0;
+
+    for rep in receiver.wait() {
```
Comment (nnmm) [inline point_viewer_grpc/src/bin/octree_benchmark.rs]: Do you know whether wait() is the right thing to do here?

**Comment by tblaette** [comment]
@SirVer can you merge? Or can we do that, too?

**Comment by SirVer** [comment]
@wally-the-cartographer merge

@tblaette you should be able to talk to wally as well.

**Comment by SirVer** [comment]
@wally-the-cartographer merge

**Pull request merged**